src/qibosoq/configuration.py

```diff
@@ -0,0 +1,39 @@
+"""Qibosoq configuration file"""
+import os
+
+HOST = "192.168.0.81"
+"""Server address"""
+if "QIBOSOQ_HOST" in os.environ:
+    HOST = os.environ["QIBOSOQ_HOST"]
```

Comment:
```suggestion
def from_env(name, default=None):
    return os.getenv(f"QIBOSOQ_{name}", default)

HOST = from_env("HOST", "192.168.0.81")
"""Server address"""
```

Reply:
In principle, we could remove even more duplication with a "dirty trick", that limited to this context would not be dirty:
```suggestion
_g = globals()
def from_env(name, default=None):
    _g[name] = os.getenv(f"QIBOSOQ_{name}", default)

from_env("HOST", "192.168.0.81")
```

But I would not recommend: generating dynamically we would lose Sphinx docs (there is no variable to assign to, since it is dynamically created) and it would be harder to understand.
I just wanted to show you that is possible (and someone else is doing similar but [better things](https://docs.pydantic.dev/latest/usage/settings/), but the first suggestion is good enough, we do not need the extra dependency, nor the extra complications),

---

You can test yourself with:
```sh
python -c "g = globals(); g['ciaocome'] = 42; print(42)"
```

Reply:
Still nice to know, thanks! (I implemented the first version, mainly for having better documentation)